docs/compiling-mono/windows/index.md

```diff
@@ -4,51 +4,104 @@ redirect_from:
   - /Compiling_Mono_on_Windows/
 ---
 
-There are two ways of building Mono on Windows, both do require a Cygwin setup which provides some tools required by Mono at build time. First, download Cygwin from [www.cygwin.com](http://www.cygwin.com).
+To build Mono on Windows, a Cygwin setup is required since it provides some tools required by Mono at build time. You'll also need Visual Studio 2015 or later installed.
 
-Run the following command in cmd.exe to install the required packages:
+## Install prerequisites
 
-``` shell
-setup-x86.exe --root "C:\cygwin" --quiet-mode --packages autoconf,automake,bison,gcc-core,gcc-g++,mingw64-i686-runtime,mingw64-i686-binutils,mingw64-i686-gcc-core,mingw64-i686-gcc-g++,mingw64-i686-pthreads,mingw64-x86_64-runtime,mingw64-x86_64-binutils,mingw64-x86_64-gcc-core,mingw64-x86_64-gcc-g++,mingw64-x86_64-pthreads,w32api-headers,libtool,make,python,gettext-devel,gettext,intltool,libiconv,pkg-config,git,curl,libxslt
+* Download and install 64 bit Cygwin from [www.cygwin.com](http://www.cygwin.com).
+* Install [Visual Studio 2015 or later](https://www.visualstudio.com) - Community Edition works fine.
+* Download and install [Mono for Windows](http://www.mono-project.com/docs/getting-started/install/windows/)
+* Run the following command in cmd.exe to install the required packages: 
+```
```

Comment:
please add a newline between the list and the backticks (it confuses the markdown renderer otherwise).